tests/translator/output/api_with_basic_custom_domain.json

```diff
@@ -165,7 +165,7 @@
     },
     "MyApifetchBasePathMapping": {
       "Properties": {
-        "BasePath": "fetch",
+        "BasePath": "/fetch",
```

Comment:
Test outputs should typically never change. Why are these changing?

Reply:
removed. Changed the code to match `Http API` where we remove trailing and leading `/`.